src/linkml_transformer/transformer/eval_utils.py

```diff
@@ -0,0 +1,199 @@
+"""
+meta-circular interpreter for evaluating python expressions
+
+ - See `<https://stackoverflow.com/questions/2371436/evaluating-a-mathematical-expression-in-a-string>`_
+"""
```

Comment:
copied this file over from `linkml_runtime` as I needed to hack the existing `eval_expr` function a bit.